src/mmw/apps/export/views.py

```diff
@@ -33,6 +36,7 @@
 SHAPEFILE_EXTENSIONS = ['cpg', 'dbf', 'prj', 'shp', 'shx']
 DEFAULT_KEYWORDS = {'mmw', 'model-my-watershed'}
 MMW_APP_KEY_FLAG = '{"appkey": "model-my-watershed"}'
+EXCEL_TEMPLATE = '/opt/app/worksheet/MMW_BMP_Spreadsheet_Tool.xlsx'
```

Comment:
Is this path portable between environments (dev and aws)? My initial instinct would be to make this resource in a directory relative to the `views.py` file. Is there precedent for either approach in the codebase?

Reply:
This path is portable, but doing something relative is probably better, and has precedent: 

https://github.com/WikiWatershed/model-my-watershed/blob/51367b833b72717d48b394cfd431cf2931453d94/src/mmw/mmw/settings/layer_settings.py#L24-L27

Will change.